src/cfn/index.ts

```diff
@@ -1358,7 +1362,12 @@ export async function updateStackMain(argv: GenericCLIArguments): Promise<number
 
     const createChangesetResult = await changeSetRunner.run();
     if (createChangesetResult > 0) {
-      return createChangesetResult;
+      if (changeSetRunner._hasChanges) {
```

Comment:
If we're going to use this property outside of the class we should remove the underscore which signals privateness.